{"uid":"bcf9afab14ab1164","name":"Создание заказа с невалидными данными","fullName":"tests.store.test_store.TestStore#test_create_order_with_invalid_data","historyId":"278cf9533d19079b37880aa5c655077e","time":{"start":1744327889283,"stop":1744327889298,"duration":15},"status":"failed","statusMessage":"AssertionError: Статус код 500 не равен ожидаемому 400","statusTrace":"self = <test_store.TestStore object at 0x7f3c7d724cb0>\nclient = <src.models.client.Client object at 0x7f3c7d72a6f0>\nstore_helper = <src.func.store.helpers.StoreHelper object at 0x7f3c7d72adb0>\norder_data = Order(id=2025008213453549843908439809804398034593454536556445, pet_id=94, quantity=10, ship_date='1996-09-13T14:19:21.392297', status='placed', complete=False)\n\n    @allure.title(\"Создание заказа с невалидными данными\")\n    def test_create_order_with_invalid_data(self, client, store_helper, order_data):\n    \torder_data.id = 2025008213453549843908439809804398034593454536556445\n>   \tresponse = store_helper.place_order(client, order_data, expected_status_code=400)\n\ntests/store/test_store.py:72: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nsrc/func/store/helpers.py:34: in place_order\n    CustomAsserts.check_status_code(response, expected_status_code)\nsrc/tech/custom_asserts.py:53: in check_status_code\n    _check_status_code(response, expected_status_code)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresp = <Response [500]>, expected_code = 400\n\n    @step(f\"Статус код = {expected_status_code}\")\n    def _check_status_code(resp, expected_code):\n>   \tassert resp.status_code == expected_code, f\"Статус код {resp.status_code} не равен ожидаемому {expected_code}\"\nE    AssertionError: Статус код 500 не равен ожидаемому 400\n\nsrc/tech/custom_asserts.py:51: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1744327868637,"stop":1744327868639,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"client","time":{"start":1744327868639,"stop":1744327868639,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"store_helper","time":{"start":1744327889040,"stop":1744327889040,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"order_data","time":{"start":1744327889281,"stop":1744327889282,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Статус код 500 не равен ожидаемому 400","statusTrace":"self = <test_store.TestStore object at 0x7f3c7d724cb0>\nclient = <src.models.client.Client object at 0x7f3c7d72a6f0>\nstore_helper = <src.func.store.helpers.StoreHelper object at 0x7f3c7d72adb0>\norder_data = Order(id=2025008213453549843908439809804398034593454536556445, pet_id=94, quantity=10, ship_date='1996-09-13T14:19:21.392297', status='placed', complete=False)\n\n    @allure.title(\"Создание заказа с невалидными данными\")\n    def test_create_order_with_invalid_data(self, client, store_helper, order_data):\n    \torder_data.id = 2025008213453549843908439809804398034593454536556445\n>   \tresponse = store_helper.place_order(client, order_data, expected_status_code=400)\n\ntests/store/test_store.py:72: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nsrc/func/store/helpers.py:34: in place_order\n    CustomAsserts.check_status_code(response, expected_status_code)\nsrc/tech/custom_asserts.py:53: in check_status_code\n    _check_status_code(response, expected_status_code)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresp = <Response [500]>, expected_code = 400\n\n    @step(f\"Статус код = {expected_status_code}\")\n    def _check_status_code(resp, expected_code):\n>   \tassert resp.status_code == expected_code, f\"Статус код {resp.status_code} не равен ожидаемому {expected_code}\"\nE    AssertionError: Статус код 500 не равен ожидаемому 400\n\nsrc/tech/custom_asserts.py:51: AssertionError","steps":[{"name":"Создание заказа","time":{"start":1744327889283,"stop":1744327889298,"duration":15},"status":"failed","statusMessage":"AssertionError: Статус код 500 не равен ожидаемому 400\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/python-pytest-api/python-pytest-api/src/func/store/helpers.py\", line 34, in place_order\n    CustomAsserts.check_status_code(response, expected_status_code)\n  File \"/home/runner/work/python-pytest-api/python-pytest-api/src/tech/custom_asserts.py\", line 53, in check_status_code\n    _check_status_code(response, expected_status_code)\n  File \"/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/python-pytest-api/python-pytest-api/src/tech/custom_asserts.py\", line 51, in _check_status_code\n    assert resp.status_code == expected_code, f\"Статус код {resp.status_code} не равен ожидаемому {expected_code}\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"Статус код = 400","time":{"start":1744327889298,"stop":1744327889298,"duration":0},"status":"failed","statusMessage":"AssertionError: Статус код 500 не равен ожидаемому 400\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/python-pytest-api/python-pytest-api/src/tech/custom_asserts.py\", line 51, in _check_status_code\n    assert resp.status_code == expected_code, f\"Статус код {resp.status_code} не равен ожидаемому {expected_code}\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[{"name":"resp","value":"<Response [500]>"},{"name":"expected_code","value":"400"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"90634dd3da6848d3","name":"Request","source":"90634dd3da6848d3.txt","type":"text/plain","size":47},{"uid":"8b3d096d020c5935","name":"Request headers","source":"8b3d096d020c5935.json","type":"application/json","size":196},{"uid":"ffaaa29c0bf30fe","name":"Request body","source":"ffaaa29c0bf30fe.json","type":"application/json","size":184},{"uid":"c46d066d32ccee98","name":"Response status code","source":"c46d066d32ccee98.txt","type":"text/plain","size":3},{"uid":"2ed503ab00d9968d","name":"Response Headers","source":"2ed503ab00d9968d.json","type":"application/json","size":357},{"uid":"c522957a3bc37925","name":"Response body","source":"c522957a3bc37925.json","type":"application/json","size":77}],"parameters":[{"name":"client","value":"<src.models.client.Client object at 0x7f3c7d72a6f0>"},{"name":"data","value":"Order(id=2025008213453549843908439809804398034593454536556445, pet_id=94, quantity=10, ship_date='1996-09-13T14:19:21.392297', status='placed', complete=False)"},{"name":"expected_status_code","value":"400"}],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":6,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":2,"attachmentsCount":6,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"Store"},{"name":"epic","value":"Petstore API"},{"name":"tag","value":"store"},{"name":"parentSuite","value":"tests.store"},{"name":"suite","value":"test_store"},{"name":"subSuite","value":"TestStore"},{"name":"host","value":"fv-az1065-663"},{"name":"thread","value":"2296-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.store.test_store"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":["store"]},"source":"bcf9afab14ab1164.json","parameterValues":[]}